web/modules/custom/joinup_core/joinup_core.info.yml

```diff
@@ -108,3 +90,6 @@ config_devel:
     - tour.tour.account
     - tour.tour.content_overview
     - tour.tour.rdf_entity_canonical
+  optional:
+    - core.entity_form_display.comment.comment.default
+    - field.field.comment.comment.field_body
```

Comment:
This is the same as below. I don't think that fields necessary to the website (meaning that body fields are vital to the entities they are attached to even if they are not semantically necessary) should be placed in the optional section just for the sake of relaxing dependencies.. This is more of a workaround in my mind than an improvement :/